network_modeling/configs/templates/1.3/aruba/common/sw-cdu.primary.j2

```diff
@@ -37,6 +37,38 @@ spanning-tree config-revision 1
 interface mgmt
     shutdown
     ip dhcp
+qos queue-profile Aruba
```

Comment:
This is the same config block for cdus and spines and leafs.  Let's  create a new template in common and reference the template here and in the other main switch templates.

Reply:
Roger dodger. Creating Jinja template for Qos

Reply:
resolved.